core/shared/src/main/scala/monocle/Lens.scala

```diff
@@ -1,6 +1,7 @@
 package monocle
 
-import scalaz.{Applicative, Choice, Functor, Monoid, Split, \/}
+import monocle.function.fields._
```

Comment:
it is a tiny details but would you mind to specify which optics do you import. I think it would help the reader: `import monocle.function.fields.{first, second}`